ARGUS/ARGUS_Transforms.py

```diff
@@ -170,34 +183,38 @@ def make_slices(smin, smax, _start, _end):
                 outstd2 = np.std(arr[tuple(slices)],axis=self.axis)
 
                 outstdMin = np.minimum(outstd0,outstd1)
-                outstdMin = np.minimum(outstdMin, outstd2)
+                np.minimum(outstdMin, outstd2, out=out_img[out_img_slice])
+                out_img_slice += 1
+
                 outstdMax = np.maximum(outstd0,outstd1)
-                outstdMax = np.maximum(outstdMax,outstd2)
-                img = np.stack([outmean, outstd, outstdMin, outstdMax])
-            else:
-                img = np.stack([outmean, outstd])
+                np.maximum(outstdMax, outstd2, out=out_img[out_img_slice])
+                out_img_slice += 1
+
             if self.include_center_slice:
                 r_min = self.num_slices // 2
                 r_max = r_min + 1
                 r_min,r_max,slices = make_slices(r_min,r_max,_start,_end)
-                rawframe = arr[tuple(slices)]
-                img = np.concatenate([img,rawframe])
+                out_img[out_img_slice] = arr[tuple(slices)]
+                out_img_slice += 1
             if self.include_mean_abs_diff:
                 framediff = np.absolute(np.diff(arr,axis=self.axis))
-                meandiff = np.mean(framediff,axis=self.axis,keepdims=True)
-                img = np.concatenate((img,meandiff),axis=self.axis)
+                np.mean(framediff,axis=self.axis,out=out_img[out_img_slice])
+                out_img_slice += 1
             if self.include_skewness:
-                skewness = sp.stats.skew(arr,axis=self.axis,keepdims=True)
-                img = np.concatenate((img,skewness),axis=self.axis)
+                out_img[out_img_slice] = sp.stats.skew(arr,axis=self.axis)
+                out_img_slice += 1
             if self.include_kurtosis:
-                kurtosis = sp.stats.kurtosis(arr,axis=self.axis,keepdims=True)
-                img = np.concatenate((img,kurtosis),axis=self.axis)
+                out_img[out_img_slice] = sp.stats.kurtosis(arr,axis=self.axis)
+                out_img_slice += 1
             if self.include_gradient:
                 gradlist = np.gradient(arr)
                 for d in range(len(gradlist)):
-                    grad = np.mean(gradlist[d],axis=self.axis,keepdims=True)
-                    img = np.concatenate((img,grad),axis=self.axis)
-        return img
+                    np.mean(gradlist[d],axis=self.axis,
+                        out=out_img[out_img_slice])
+                    out_img_slice += 1
+            return out_img
+
+        return arr

```

Comment:
Thanks for adding these additional stats. Looking forward to trying additional training variants using these.